rskj-core/src/test/java/co/rsk/peg/federation/FederationTestUtils.java

```diff
@@ -46,6 +48,36 @@ public final class FederationTestUtils {
     private FederationTestUtils() {
     }
 
+    public static final List<BtcECKey> TEST_GENESIS_FED_SIGNERS = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+        new String[]{"fa01", "fa02", "fa03", "fa04", "fa05", "fa06", "fa07", "fa08", "fa09"}, true
+    );
+
+    public static List<BtcECKey> TEST_GENESIS_ERP_SIGNERS = BitcoinTestUtils.getBtcEcKeysFromSeeds(
```

Comment:
Do these need to be public?

Reply:
I put it as public just in case they are need it somewhere else to test signing a transaction

Reply:
If it's not needed then I wouldn't make it public. Public keys can be obtained from the federation object

Reply:
Done